sdk/eventhubs/azure-messaging-eventhubs-stress/README.md

```diff
@@ -0,0 +1,300 @@
+...
+annotations:
+  namespace: <stress test namespace>
+```
+
+Note: the default value of stress test namespace is `java-eh`. You can keep it when no other stress test are running.
+
+Download project [azure-sdk-tools][azure_sdk_tools] for the deployment to stress test cluster.
+
+Keep in current project path, run below command:
+
+```shell
+<root path>\azure-sdk-tools\eng\common\scripts\stress-testing\deploy-stress-tests.ps1 -PushImage -Login -Namespace <stress test namespace>
+```
+
+### Validate status
+
+Get pods and jobs:
+
+```shell
+helm list -n <stress test namespace>
+kubectl get pods -n <stress test namespace>
+kubectl get jobs -n <stress test namespace>
+```
+
+List stress test pods:
+
+```shell
+kubectl get pods -n <stress test namespace> -l release=<stress test name>
+```
+
+Get logs from the init-azure-deployer init container, if deploying resources. Omit `-c init-azure-deployer` to get main container logs.
+
+```shell
+kubectl logs -n <stress test namespace> <stress test pod name> -c init-azure-deployer
+```
+
+If above command output is empty, there may have been startup failures:
+
+```shell
+kubectl describe pod -n <stress test namespace> <stress test pod name>
+```
+
+Get stress test logs:
+
+```shell
+kubectl logs -n <stress test namespace> <stress test pod name>
+# Note that we may define multiple containers (for example, sender and receiver)
+kubectl logs -n <stress test namespace> <stress test pod name> -c <container name>
+```
+
+Stop and remove deployed package:
+
+```shell
+helm uninstall <stress test name> -n <stress test namespace>
+```
+
+### Configure Faults
+
+[Chaos Mesh](https://chaos-mesh.org/) is used to configure faults against test jobs. There are two ways for the configuration, which are via the UI or via kubernetes manifests.
+
+#### Chaos Dashboard
+
+Make sure you can access the chaos dashboard by running the below command, and navigating to localhost:2333 in your browser.
+
+```shell
+kubectl port-forward -n stress-infra svc/chaos-dashboard 2333:2333
+```
+From dashboard, click `New experiment` and choose your parameters to submit a fault experiment.
+
+#### Chaos Manifest
+
+See [Chaos manifest](https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/chaos/README.md#chaos-manifest) for details.
+
+
+### Configure Monitor
+
+Download [application insights java agent][java_agent_jar]. 
+
+Place the `applicationinsights-agent-3.2.11.jar` under `src\main\resources` folder.
+
+Add JVM parameter `-javaagent:src\main\resources\applicationinsights-agent-3.2.11.jar` for java command in `job.yaml`.
+```yaml
+java -javaagent:BOOT-INF/classes/applicationinsights-agent-3.2.11.jar \
+"org.springframework.boot.loader.JarLauncher" \
+```
+
+
+Redeploy job to cluster and monitor application insights under resource group `rg-stress-cluster-test`.
+
+
+## Key concepts
+
+The stress tests for event hubs client is developed from [azure-sdk-chaos][azure_sdk_chaos].
+
+### Project Structure
```

Comment:
Same comment here, I think it would be better to link the main docs: https://github.com/Azure/azure-sdk-tools/blob/main/tools/stress-cluster/chaos/README.md#layout

Reply:
For layout, I am wondering if chaos changes the layout or has some updates, there may be some gaps for project layout until we update this project accordingly. Can we keep this part to reflect current project's layout? I can add the additional link to the main docs for reference.

Reply:
For layout, I am wondering if chaos changes the layout or has some updates, there may be some gaps for project layout until we update this project accordingly. Can we keep this part to reflect current project's layout? I can add the additional link to the main docs for reference.

Reply:
Ah I see, that makes sense. Maybe both? Add a link to the canonical docs at the top of the section while also outlining the docs on the current java layout.

Reply:
Sure, I agree to do this way.